### PR TITLE
Feature/asset classes docs

### LINKED ISSUE #55: Feature/add ignore rules for asset-classes
## **Is your feature request related to a problem? Please describe.**
Generated asset-classes come up with non documented members and class being public ones
https://dart-lang.github.io/linter/lints/public_member_api_docs.html
https://dartcodemetrics.dev/docs/rules/common/member-ordering-extended

## **Describe the solution you'd like**
Add ignore rules

## **Describe alternatives you've considered**
Generate docs for both members and class. ( I think it's possible to come up with docs to class and impossible to add logical docs for members, so i decided that ignore rules are enough)

### PR DESCRIPTION
**Pull request by Sanlovty**
### Description of the Change
Start asset-class file generation with ignore_rules

### Benefits
Follow linter rules ,no more warnings

### Verification Process
Generate asset-classes and analyze with linter

### Applicable Issues
Closes #55

### CODE REVIEW

**Comment by Sanlovty** [comment]
**Update:** I can make this solution more interesting - we can allow user to pass the set of rules [first-rule, second-rules, third-rule] instead of hardcoding it ( sometimes user can set linter rules that include member sorting, so we will prevent this)

File: lib/src/data/export_template.dart [hunk 1]
```diff
@@ -13,6 +13,10 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+/// A template to add ignore rules to prevent the warnings from
+/// [member-ordering-extended] and [public_member_api_docs] rules.
+String get ignoreRulesTemplate =>
+    '// ignore_for_file: public_member_api_docs, member-ordering-extended\n\n';
```
Comment (BirjuVachhani) [inline lib/src/data/export_template.dart:19]: `member-ordering-extended` is not from official Flutter/Dart lints. We shouldn't add that I believe. Besides, I thought it was ordering references by name, wasn't it?

**Review by BirjuVachhani — CHANGES_REQUESTED** [review]

File: lib/src/data/export_template.dart (hunk 1 above)
Comment (Sanlovty) [reply lib/src/data/export_template.dart:19]: I added the idea to issue. We can allow user do add rules  list user want to be ignored in files

Comment (BirjuVachhani) [reply lib/src/data/export_template.dart:19]: I like that idea!

Comment (BirjuVachhani) [reply lib/src/data/export_template.dart:19]: This PR is redundant then, isn't it?

Comment (Sanlovty) [reply lib/src/data/export_template.dart:19]: Yes, it was made before the idea. I'l make normal one later(tomorrow)

Comment (BirjuVachhani) [reply lib/src/data/export_template.dart:19]: Alright then, closing this PR for now.

**Pull request closed without merging**